Error while checking build/O2/fullCI_slc9 for 45c6981d03730d999f74f6e5c3c8b90d8acf4348 at 2025-12-16 16:05:
```
## sw/BUILD/O2-latest/log
/sw/BUILD/e18792e2e5b0029e26ceb633a1248b1380ac7e0a/O2/Detectors/TPC/base/G__O2TPCBase.cxx:10555:84: error: 'class ROOT::TGenericClassInfo' has no member named 'AdoptMemberStreamer'; did you mean 'AdoptStreamer'?
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/14930/45c6981d03730d999f74f6e5c3c8b90d8acf4348/build_O2_fullCI_slc9/pretty.html).

--- a/Detectors/TPC/base/src/TPCFlagsMemberCustomStreamer.cxx
+++ b/Detectors/TPC/base/src/TPCFlagsMemberCustomStreamer.cxx
@@ -71,14 +71,8 @@ namespace ROOT
 {
 static __attribute__((used)) int _R__dummyStreamer_3 =
   ([]() {
-    auto cl = TClass::GetClass<o2::tpc::CalArray<o2::tpc::PadFlags>>();
-    if (cl) {
-      if (!getenv("TPC_PADFLAGS_STREAMER_OFF")) {
-        cl->AdoptMemberStreamer("mData", new TMemberStreamer(MemberVectorPadFlagsStreamer));
-      }
-    } else {
-      // we should never come here ... and if we do we should assert/fail
-      assert(false);
+    if (!getenv("TPC_PADFLAGS_STREAMER_OFF")) {
+      ROOT::GenerateInitInstance((o2::tpc::CalArray<o2::tpc::PadFlags> *)nullptr)->AdoptMemberStreamer("mData", new TMemberStreamer(MemberVectorPadFlagsStreamer));
     }
     return 0;
   })();

